manifests/n/NordicSemiconductor/nrfutil/7.6.0/NordicSemiconductor.nrfutil.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: NordicSemiconductor.nrfutil
+PackageVersion: 7.6.0
+InstallerType: portable
+Commands:
+- nrfutil

```

Comment:
```suggestion
- nrfutil
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

Reply:
Thanks @stephengillie, hopefully it goes through now 👍